tests/test_xopen.py

```diff
@@ -2,35 +2,37 @@
 Tests for the xopen.xopen function
 """
 import bz2
-import subprocess
-import sys
-import tempfile
-from contextlib import contextmanager
 import functools
 import gzip
 import io
 import lzma
 import os
-from pathlib import Path
 import shutil
+import subprocess
+import sys
+import tempfile
+from contextlib import contextmanager
+from pathlib import Path
```

Comment:
I agree having sorted imports is nicer, but this should be done in a separate PR. (It’s ok this time, just leave it as it is.)